localization/english/wc_game_concepts_l_english.yml

```
@@ -50,3 +50,9 @@
  
  game_concept_necro_government:0 "Necro Government"
  game_concept_necro_government_desc:0 "A [dark_government|E] available to [GetTrait('being_undead').GetName( GetNullCharacter )] with $doctrine_death_magic_name$ accepted. Necro [armies|E] reinforce much faster."
+ 
+ game_concept_twisting_nether:0 "Twisting Nether"
+ game_concept_twisting_nether_desc:0 "Instead of [character_death_dying|E], [characters|E] with [GetTrait('being_demon').GetName( GetNullCharacter )] [trait|E] return to the Twisting Nether.\n\nThey get [GetTrait('in_twisting_nether').GetName( GetNullCharacter )] [trait|E] but can return to Azeroth some time later.\n\nReturning increase [stress|E], decreases [prestige_level|E].\n\nDemons can't return if they...\n$EFFECT_LIST_BULLET$Vanish\n$EFFECT_LIST_BULLET$Are sacrificed\n$EFFECT_LIST_BULLET$Suicide\n$EFFECT_LIST_BULLET$Are scared to death"
```

```suggestion
 game_concept_twisting_nether_desc:0 "Instead of [character_death_dying|E], [characters|E] with [GetTrait('being_demon').GetName( GetNullCharacter )] [trait|E] returns to the Twisting Nether.\n\nThey get [GetTrait('in_twisting_nether').GetName( GetNullCharacter )] [trait|E] but can return to Azeroth some time later.\n\nReturning increase [stress|E], decreases [prestige_level|E].\n\nDemons can't return if they...\n$EFFECT_LIST_BULLET$Vanish\n$EFFECT_LIST_BULLET$Are sacrificed\n$EFFECT_LIST_BULLET$Suicide\n$EFFECT_LIST_BULLET$Are scared to death"
```
I assume ``[GetTrait('being_demon').GetName( GetNullCharacter )] [trait|E]`` is a third person, singular subject (He, She, It). Thus it should be "returns".

---

Nope, it's `Characters with Demon Trait return`.

---

Alright, thanks for clarifying